lib/rocky-lint.js

```diff
@@ -46,6 +46,16 @@ const WARN = makeLogFilter(0);
 const INFO = makeLogFilter(1);
 const DEBUG = makeLogFilter(2);
 
+(function checkNodeVersion() {
+  const semver = require('semver');
+  const nodeVersionString = process.versions.node;
+  const minimumVersionString = '5.10.0';  // because we're using fs.mkdtemp
+  if (!semver.satisfies(nodeVersionString, '>=' + minimumVersionString)) {
+    WARN(`Node ${minimumVersionString} is required, but ${nodeVersionString} is used.`);
```

Comment:
We don't have a CI setup which would have told you: this line violates the max-line-length linter rule.